schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -806,6 +806,20 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-12-01/Microsoft.Network.json#/resourceDefinitions/applicationGateways" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-12-01/Microsoft.Network.json#/resourceDefinitions/expressRouteCircuits_authorizations" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-12-01/Microsoft.Network.json#/resourceDefinitions/ExpressRoutePorts" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-02-01/Microsoft.Network.json#/resourceDefinitions/ddosProtectionPlans" },
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2019-02-01/Microsoft.Network.json#/resourceDefinitions/ddosProtectionPlans" }, [](start = 18, length = 133)

we have a new root schema now: 2019-04-01/deploymentTemplate.json. Can you please add them to new root schema as well?

Reply:
updated